2-Draft-Accepted/RFCNNNN-Support-Experimental-Features.md

```diff
@@ -327,8 +337,11 @@ For experimental features in PowerShell engine,
 an internal enum `EngineExperimentalFeatures` will be used to track the names of all available engine experimental features.
 When adding a new experimental feature, its name needs to be added to the enum for the feature to be discoverable to users.
 The feature description needs to be added as a resource string,
-and the id of the resource string needs to be constructible from the feature name,
+and the id of the resource string needs to be constructed from the feature name,
 such as `XXX-Description` where `XXX` is the feature name.
+
+> NOTE: Do experimental features really need resource strings for the description?  They are not likely to be localized and it will simplify declaration since the name and description can be placed in a table as a single unit and avoid misnamed or missing description resources.
```

Comment:
probably not - it's probably ok not to require this.

Reply:
> as a single unit and avoid misnamed or missing description resources.

I think that make senses. Then the enum could really be a static table instead.

Reply:
Updated the PR to use a static read-only collection to track all engine experimental features.